Updating headers + tracing project CTAs (#892)
langchain-ai/langsmith-docs: docs/observability/how_to_guides/online_evaluations.mdx

- **[LLM-as-a-judge](/evaluation/concepts#llm-as-judge)**: Use an LLM to evaluate your traces. Used as a scalable way to provide human-like judgement to your output (e.g. toxicity, hallucination, correctness, etc.).
- **Custom Code**: Write an evaluator in Python directly in LangSmith. Often used for validating structure or statistical properties of your data.

## View online evaluators

Head to the **Tracing Projects** tab and select a tracing project. To view existing online evaluators for that project, click on the **Evaluators** tab.

![View online evaluators](./static/view_evaluators.png)

## Configure online evaluators

#### 1. Navigate to online evaluators

> Head to the **Tracing Projects** tab and select a tracing project. Click on **+ New** in the top right corner the tracing projectpage, then click on **New Evaluator**.

Reviewer: ```suggestion
Head to the **Tracing Projects** tab and select a tracing project. Click on **+ New** in the top right corner of the tracing project page, then click on **New Evaluator**.
```

#### 2. Name your evaluator

#### 3. Create a filter

For example, you may want to apply specific evaluators based on:

> - Runs where a [user left feedback](/evaluation/how_to_guides/attach_user_feedback) indicating he response was unsatisfactory
> - Runs that invoke a specific tool call. See [filtering for tool calls](//observability/how_to_guides/filter_traces_in_application#example-filtering-for-tool-calls) for more information

Reviewer: ```suggestion
- Runs where a [user left feedback](/evaluation/how_to_guides/attach_user_feedback) indicating the response was unsatisfactory.
- Runs that invoke a specific tool call. See [filtering for tool calls](//observability/how_to_guides/filter_traces_in_application#example-filtering-for-tool-calls) for more information.
```

- Runs that match a particular piece of metadata (eg. if you log traces with a plan_type and only want to run evaluations on traces from your enterprise customers). See [adding metadata to your traces](/observability/how_to_guides/add_metadata_tags) for more information.

> Filters on evaluators work the same way as filters applied traces in the project. For more information on filters, you can refer to [this guide](./filter_traces_in_application).

Reviewer: ```suggestion
Filters on evaluators work the same way as filters applied to traces in the project. For more information on filters, you can refer to [this guide](./filter_traces_in_application).
```

#### 5. (Optional) Apply rule to past runs

Apply rule to past runs by toggling the **Apply to past runs** and entering a "Backfill from" date. This is only possible upon rule creation. Note: the backfill is processed as a background job, so you will not see the results immediately.

In order to track progress of the backfill, you can view logs for your evaluator by by heading to the **Evaluators** tab within a tracing project and clicking the Logs button for the evaluator you created. Online evaluator logs are similar to [automation rule logs](./rules#view-logs-for-your-automations).

- Add an evaluator name
- Optionally filter runs that you would like to apply your evaluator on or configure a sampling rate.
- Select **Apply Evaluator**

#### 6. Select evaluator type

- Configuring [LLM-as-a-judge evaluators](/observability/how_to_guides/online_evaluations#configure-a-llm-as-a-judge-online-evaluator)
- Configuring [custom code evaluators](/observability/how_to_guides/online_evaluations#configure-a-custom-code-evaluator)

### Configure a LLM-as-a-judge online evaluator

> View this guide to configure on configuring an [LLM-as-a-judge evaluator](/evaluation/how_to_guides/llm_as_judge?mode=ui).

Reviewer: ```suggestion
View this guide to configure an [LLM-as-a-judge evaluator](/evaluation/how_to_guides/llm_as_judge?mode=ui).
```
